tkg/client/machine_deployment_test.go

```diff
@@ -1681,4 +1685,198 @@ var _ = Describe("Machine Deployment", func() {
 			})
 		})
 	})
+	Context("SetMachineDeployment", func() {
```

Comment:
This probably belongs in machine_deployment_cc_test.go

Reply:
We made changes to machine_deployment.go  and testing the whole flow, hence no test required in machine_deployment_cc_test.go

Reply:
My comment was more that this just has a context for the clusterclass version, so if the test were moved into `machine_deployment_cc_test.go` then you could remove one level of nesting from the test hierarchy.

Reply:
Makes sense, I will make the change.